scripts/start.sh

```diff
@@ -2,7 +2,7 @@
 echo "---Ensuring UID: ${UID} matches user---"
 usermod -u ${UID} ${USER}
 echo "---Ensuring GID: ${GID} matches user---"
-groupmod -g ${GID} ${USER} 
+groupmod -g ${GID} ${USER} >> /dev/null ||:
```

Comment:
Tested it again and got the same:
```
root@e4a8dda95449:/# groupmod -g ${GID} ${USER} >> /dev/null ||:
groupmod: GID '100' already exists
```

Shouldn't it be: `groupmod -g ${GID} ${USER} 2>> /dev/null ||:`